### PR TITLE
Rename Chargeads to Newsroom AI

### PR DESCRIPTION
**Pull request by danielstockton**
Part of a rebranding effort.
Old `chargeads` function left in place, to not break existing tags.

♻️ Refactoring (like moving around code w/o any changes) (`:recycle:`)

### CODE REVIEW

**Comment by torch2424** [comment]
Triaiging to @zhouyx 😄 , Feel free to re-assign.

My only concern is, what about the amp pages that currently have `<amp-ad type="chargeads">` ? 🤔 

Thank you for your contribution by the way! 👍

**Comment by zhouyx** [comment]
Thank you @torch2424, @danielstockton I have the exact same question. What do you expect the migration process to look like? Thank you

**Comment by danielstockton** [comment]
We will be informing people that they should replace their tags, as the old ones will be deprecated.
After we have confirmed that there are no longer any old tags, we will make another commit to completely remove chargeads.

File: ads/_config.js [hunk 1]
```diff
@@ -358,8 +358,6 @@ export const adConfig = {
     renderStartImplemented: true,
   },
 
-  'chargeads': {},
```
Comment (zhouyx) [inline ads/_config.js]: You'll still need to keep the config here to be backward compatible. Could you please leave a comment wherever there's `chargeads` with an estimated date of deprecation. Thank you.

**Comment by zhouyx** [comment]
I see thank you @danielstockton. Could you please provide us with an estimate date of deprecation?

File: ads/_config.js (hunk 1 above)
Comment (danielstockton) [reply ads/_config.js]: Updated, will submit another PR in a month to remove chargeads

File: ads/_config.js [hunk 2]
```diff
@@ -358,7 +358,7 @@ export const adConfig = {
...
+  'chargeads': {},  # Deprecated, to be removed on 2019-04-06
```
Comment (zhouyx) [inline ads/_config.js]: Thank you for updating the date. 
not python : ) `//` to add comment 😄

Comment (danielstockton) [reply ads/_config.js]: Sorry :man_facepalming:

File: 3p/integration.js [hunk 3]
```diff
@@ -188,6 +187,7 @@ import {nend} from '../ads/nend';
 import {netletix} from '../ads/netletix';
 import {noddus} from '../ads/noddus';
 import {nokta} from '../ads/nokta';
+import {chargeds, nws} from '../ads/nws';
```
Comment (zhouyx) [inline 3p/integration.js]: nit: chargeads

Comment (danielstockton) [reply 3p/integration.js]: Sorry, corrected

File: ads/_config.js [hunk 4]
```diff
@@ -358,7 +358,7 @@ export const adConfig = {
     renderStartImplemented: true,
   },
 
-  'chargeads': {},
+  'chargeads': {},  // Deprecated, to be removed on 2019-04-06
```
Comment (zhouyx) [inline ads/_config.js]: nit: to fix the lint check, please keep one single space before `//`

**Comment by danielstockton** [comment]
@zhouyx Sure, updated the example. Thanks.

**Comment by zhouyx** [comment]
@danielstockton Could you please fix the lint error so that I can merge the change. Thank you

Screen Shot 2019-04-02 at 6 49 46 PM

**Comment by danielstockton** [comment]
Can you please let us know if anything else is preventing this from being merged? Thanks

**Comment by zhouyx** [comment]
@danielstockton The travis build is still red to me, which prevents me from merging it. I've restarted it to see if it's a flake.

**Comment by zhouyx** [comment]
Travis build green. PR merged. Thank you!

**Comment by danielstockton** [comment]
Thanks!

**Comment by rsimha** [comment]
@zhouyx This PR has broken `master`: https://travis-ci.org/ampproject/amphtml/jobs/532077068#L314

Seems like there was a conflicting change that caused Travis checks to pass earlier.

**Pull request merged**